src/utils/forApi.js

```diff
@@ -0,0 +1,59 @@
+import axios from "axios";
+
+export const baseurl = "https://fin-tube.herokuapp.com";
+// export const baseurl = "http://127.0.0.1:8080";
+
+export const deletePlaylistHandler = async (playlistName, token, dispatch) => {
+  const response = await axios.delete(`${baseurl}/playlist/${playlistName}`, {
+    headers: { authorization: token },
+  });
+  if(response.status === 200){
+    dispatch({ type: "REMOVE_PLAYLIST", payload: playlistName })
+  }
+};
+
+export const deleteVideoHandler = async (playlistName, videoId, token,dispatch) => {
+  try {
+    const response = await axios.delete(
+      `${baseurl}/playlist/${playlistName}/${videoId}`,
+      { headers: { authorization: token } }
+    );
+    if(response.status === 200){
+      dispatch({type: "REMOVE_FROM_PLAYLIST", payload:{playlistName, videoId}})
+    }
+  } catch (error) {
+    console.log(error.response);
```

Comment:
it will be good to have better error management for all API requests rather than logging to the console. eg: showing an error toast.